OracleUnifiedDirectory/README.md

```diff
@@ -0,0 +1,134 @@
+
+This project offers Dockerfiles and scripts to build and configure an Oracle Unified Directory image based on 12cPS4 (12.2.1.4.0) release. Use this Docker Image to facilitate installation, configuration, and environment setup for DevOps users. 
+
+This Image refers to binaries for OUD Release 12.2.1.4.0 and it has the capability to create different types of OUD Instances (Directory Service, Proxy, Replication) on containers targeted for development and testing.
+
+***Image***: oracle/oud:12.2.1.4.0
+
+## Prerequisites for OUD on Docker
+
+The following prerequisites are applicable when using the OUD Docker image on Docker only:
+
+* A working installation of Docker 18.03 or later
+
+## Prerequisites for OUD on Kubernetes
+
+Refer to the [Prerequisites](https://oracle.github.io/fmw-kubernetes/oud/prerequisites) in the Oracle Unified Directory Kubernetes documentation.
+
+## Installing the OUD Docker Image
+
+An OUD Docker image can be created and/or made available for deployment in a Docker environment in the following ways:
+
+1. Download a pre-built OUD Docker image from [My Oracle Support](https://support.oracle.com) (Patch 31979398).  This image contains the latest bundle patch and one-off patches for Oracle Unified Directory 12.2.1.4.0.
+2. Build your own OUD Docker image using the WebLogic Image Tool. Oracle recommends using the Weblogic Image Tool to build your own OUD 12.2.1.4.0 image along with the latest Bundle Patch and any additional patches that you require. For more information, see [Building an OUD Docker Image using Image Tool](https://github.com/oracle/docker-images/OracleUnifiedDirectory/imagetool/12.2.1.4.0)
+
+## Running the OUD Docker Image in a Docker Container
+
+The OUD Docker Image supports running the following services in a Docker container:
+
+*  Directory Server/Service [instanceType=Directory]
+*  Directory Proxy Server/Service [instanceType=Proxy]
+*  Replication Server/Service [instanceType=Replication]
+*  Directory Server/Service added to existing Directory or Replication Server/Service [instanceType=AddDS2RS]
+*  Directory Client to run CLIs like ldapsearch, dsconfig, and dsreplication.
+
+The functionality and features available from the OUD image will depend on the environment variables passed when setting up/starting the container. Configuration of instances with support for Proxy and Replication require invocation of dsconfig and dsreplication commands following the execution of oud-setup. The OUD 12c Docker image is designed to support passing dsconfig and dsreplication parameters as required, to be used with commands after instance creation using oud-setup or oud-proxy-setup.  This provides flexibility in the types of OUD service that can be configured to run in a Docker container.
```

Comment:
Replace the word "Docker" with "container" and use backticks to format words like `dsconfig` and `dsreplication` in a monospace font.